test/runtests.jl

```diff
@@ -66,13 +68,13 @@ let
     prob = DAEProblem(resrob,u0,du0,(0.0,100000.0))
     dt = 1000
     saveat = float(collect(0:dt:100000))
-    sol = solve(prob, daskr())
-    sol = solve(prob, daskr(),save_timeseries=false)
-    sol = solve(prob, daskr(), saveat = saveat, isdiff = [true, true, false])
-    sol = solve(prob, daskr(), saveat = saveat,
+    sol = solve(prob, DASKR.Algorithm())
```

Comment:
For proper capitalization. `daskr` is/was a type constructor. I don't like `Daskr`. `DASKRAlg` or `DASKRAlgorithm` don't look as readable to me as `DASKR.Algorithm`, particularly since types are shown with the module often (`DASKR.DASKRAlg`).